src/ts/utils/roam.ts

```diff
@@ -44,8 +44,14 @@ export const Roam = {
         return Promise.reject('We\'re currently not inside roam block');
     },
 
-    async editBlock(element: HTMLElement) {
-        return Mouse.leftClick(element,20)
+    async activateBlock(element: HTMLElement) {
+        if (
+            element.classList.contains('roam-block') || 
+            element.tagName.toLocaleLowerCase() === 'textarea'
+        ) {
+            await Mouse.leftClick(element,20)
+        } 
+        return this.getRoamBlockInput();
```

Comment:
um, I'm not entirely sure I understand this comment. What I meant in the comment above is - if the passed element is the textarea - then presumably it's already "activated" and there is no need for us to do so, so that condition is not needed

Reply:
You're right. That part was redundant since `getRoamBlockInput` already does that.